timescaledb/tutorials/aws-lambda/continuous-deployment.md

```diff
@@ -0,0 +1,193 @@
+# Lambda continuous deployment with Github Actions
+
+This tutorial builds a continuous deployment (CD) pipeline between Github and AWS Lambda using Github Actions.
```

Comment:
```suggestion
This tutorial builds a continuous deployment (CD) pipeline between GitHub and AWS Lambda using GitHub Actions.
```
Really, anywhere we reference it, this is the proper capitalization. You can do the rest. 😃

Reply:
Also, `actions` should be lowercased.

Reply:
https://github.com/timescale/docs/pull/418/commits/58f1063e6f0828653ca36fed25de09b63b54bbae